templates/financial_info/edit.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}New Booking{% endblock %}
+{% block header %}
+{% endblock header %}
+
+{% block main %}
+<div class="generalInfo">
+
+  {% block formTitle %}Edit booking # {{ booking.reference }} {% endblock formTitle %}
+
+  {% block generalInfo %}
+  {{ include('financial_info/_form.html.twig', {'button_label': 'Update'}) }}
+  {% endblock generalInfo %}
```

Comment:
The edit.html.twig is a final template.
It should not create new block as no template will fill them